vagen/envs/webarena/llms/providers/openai_utils.py

```
@@ -0,0 +1,298 @@
+        
+    response = client.completions.create(
+        prompt=prompt,
+        model=model,
+        temperature=temperature,
+        max_tokens=max_tokens,
+        top_p=top_p,
+        stop=[stop_token],
+    )
+    try:
+        answer: str = response["choices"][0]["text"]
+    except:
```

Ruff (0.15.10)

[error] Do not use bare `except`

(E722)